### ISSUE TITLE
labels for edit buttons should be bilingual

### ISSUE THREAD

**Issue by sastels** [cds-snc/report-a-cybercrime#1137]
## Summary

the aria-labels for the Edit buttons on the confirmation page are not bilingual. The code supports this (The `EditButton` component) but we need to ensure that the `label` prop is marked for lingui and has a French and English text.

## Steps to reproduce

look at the html for the confirmation page (with browser inspector). the language switcher button does not switch the aria label.

## Unresolved questions

none

**Comment by naomiseminega** [comment]
Fixed by PR#1190
